Makefile

```diff
@@ -92,6 +92,38 @@ client/contracts/truffle/node_modules:
 client/build/bin/abigen:
 	cd client; build/env.sh go run build/ci.go install ./cmd/abigen
 
+go-generate: moq go-bindata stringer gencodec mockery ensure-notifications ensure-wallet-backend
+	go get -u github.com/golang/protobuf/protoc-gen-go
+	go generate ./client/cmd/control/
+	go generate ./client/cmd/faucet/
+	go generate ./client/core/
+	go generate ./client/core/types/
+	go generate ./client/core/vm/
+	go generate ./client/internal/jsre/deps/
+	go generate ./client/knode/
+	go generate ./client/knode/tracers/internal/tracers/
+	go generate ./client/p2p/discv5/
+	go generate ./notifications/blockchain/
+	go generate ./notifications/environment/
+	go generate ./notifications/keyvalue/
+	go generate ./notifications/notifier/
+	go generate ./notifications/protocolbuffer/
+	go generate ./wallet-backend/protocolbuffer/
```

Comment:
I think we'll easily forget to keep this list up to date. What if we do like in the `bindings` task?

Something like:
```
egrep -ir --exclude-dir=vendor --exclude-dir=_workspace --include \*.go "//go:generate" . | grep -v client/contracts | sed -E 's/^([^\:]*).*$/\1/' | uniq
```

Reply:
actually, if we combine this with the bindings, we can just `go generate ./...`

Reply:
go generate fails if bindings dont happen before the other, at least this was what I was getting on the beginning.

Reply:
I have been trying the approach you gave me, but the egrep version of the ci does not have that options. If you can help me in this case? I am not as good in bash!

@acroca